BritneyS/SparkleSpin
# Player screen arch

## What you did :question:
This PR is for adding the views and functionality for the 'Add Players!' screen: Adding and deleting players on the screen. The unit tests for the PlayerViewModel are also updated.

## How to test it :microscope:
1. Checkout out this branch
2. Run the app
3. Tap the 'Start' button
4. Note that the 'Add Players' screen appears, with the cursor in the top text field, and the 'Done' button greyed out
5. Type anything in the text field to represent a player name, and add the player by tapping the + button
6. Note that a cell with the player name is added to the tableview below, and the 'Done' button is no longer greyed out
7. Swipe the cell to delete the user

## Any background context you want to provide? :question:
I want to limit the number of characters a user can use in the textfield in a future PR.


## Screenshots (if applicable) :camera:
![sparklespin-pr-player-screen-arch](https://user-images.githubusercontent.com/8409475/55673510-c9690900-5876-11e9-95f4-eb05e3274507.gif)
